src/imagery/i.eodag/i.eodag.py

```diff
@@ -879,11 +879,7 @@ def skip_existing(output, search_result):
             if scene_file.exists():
                 creation_time = datetime.utcfromtimestamp(os.path.getctime(scene_file))
                 ingestion_time = scene.properties.get("modificationDate")
-                if (
-                    ingestion_time
-                    and datetime.fromisoformat(ingestion_time).replace(tzinfo=None)
-                    <= creation_time
-                ):
+                if normalize_time(ingestion_time) <= creation_time:
```

Comment:
There is some cases where `ingestion_time` woud be `None`, e.g. if you use the "onda" provider to search for S2_MSI_L2A products there will be no modificationDate parameter. So it is necessary to check if it is None or not before attempting to normalize.

If you run the following for example there won't be `modificationDate`. 
`i.eodag -j provider=onda producttype=S2_MSI_L2A output="Onda_S2" limit=1 start=2024-12-20`

```suggestion
                if ingestion_time and normalize_time(ingestion_time) <= normalize_time(
                    creation_time
                ):
```